### PR TITLE
Add waiting text checks between event click link

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?
Add waiting steps between click during the onboarding waiting around event tab.
During the BV, some of the refresh methods were starting before the page loaded ( same than https://github.com/uyuni-project/uyuni/pull/7371 )

## Links
Related to : https://github.com/SUSE/spacewalk/issues/22175

## Changelogs

- [x] No changelog needed

### CODE REVIEW

File: testsuite/features/step_definitions/common_steps.rb [hunk 1]
```diff
@@ -142,12 +142,16 @@
   # same name in the events history - however, that's the best we have so far.
   steps %(
     When I follow "Events"
+    And I wait until I see "Pending Events" text
     And I follow "Pending"
-    And I wait at most #{pickup_timeout} seconds until I do not see "#{event}" text, refreshing the page
+    And I wait until I see "Pending Events" text
+    And I wait at most 180 seconds until I do not see "#{event}" text, refreshing the page
```
Comment (vandabarata) [inline testsuite/features/step_definitions/common_steps.rb:148]:
```suggestion
    And I wait at most #{pickup_timeout} seconds until I do not see "#{event}" text, refreshing the page
```

**Review by vandabarata — APPROVED** [review]

**Review by nodeg — APPROVED** [review]
LGTM. Thank you Maxime!

**Review by elariekerboull — APPROVED** [review]

File: testsuite/features/step_definitions/common_steps.rb [hunk 2]
```diff
@@ -142,12 +142,16 @@
   # same name in the events history - however, that's the best we have so far.
   steps %(
     When I follow "Events"
+    And I wait until I see "Pending Events" text
```
Comment (srbarrios) [inline testsuite/features/step_definitions/common_steps.rb:145]: This first step might raise a timeout depending the state and the tab you are isn't it?

Comment (maximenoel8) [reply testsuite/features/step_definitions/common_steps.rb:145]: From my tests, clicking on Events tab always bring you to Pending sub-tab. I will keep an eye on this

**Pull request merged**